src/components/Search/SearchList.tsx

```diff
@@ -362,6 +326,10 @@ function SearchList(
     const selectAllButtonVisible = canSelectMultiple && !SearchTableHeader;
     const isSelectAllChecked = selectedItemsLength > 0 && selectedItemsLength === flattenedTransactionWithoutPendingDelete.length;
 
+    const keyExtractor = useCallback((item: SearchListItem, index: number) => {
+        return item.keyForList ?? `${index}`;
+    }, []);
```

Comment:
I will surprise you - this function _only_ uses it's own arguments - it can be defined completely outside of component and no need for `useCallback` 😄